### ISSUE TITLE
zeropadding conv2d or samepadding conv2d

### ISSUE THREAD

**Issue by jiangwx** [JonathanCMitchell/mobilenet_v2_keras#5]
Thank you very much for your great work!
However, one thing confuses me. 
```
from keras.applications.mobilenet_v2 import MobileNetV2
model = MobileNetV2()
model.summary()
````
Then I can get
```
Layer (type)                    Output Shape         Param #     Connected to                     
input_1 (InputLayer)            (None, 224, 224, 3)  0                                            
Conv1_pad (ZeroPadding2D)       (None, 225, 225, 3)  0           input_1[0][0]   
Conv1 (Conv2D)                  (None, 112, 112, 32) 864         Conv1_pad[0][0]  
...                 
```
It seems that conv1 adapts zeropadding. 
However, in your provided `mobilenetv2.py`, I saw
```
 first_block_filters = _make_divisible(32 * alpha, 8)
    x = Conv2D(first_block_filters,
               kernel_size=3,
               strides=(2, 2), padding='same',
               use_bias=False, name='Conv1')(img_input)
```
It seems that conv1 performs samepadding.
I want to know which padding did you adapt in the pretrained .h5 file?

**Comment by jiangwx** [comment]
I find that 'same padding' is equilevant to 'zeropadding' in keras 2.2.4.

**Comment by JonathanCMitchell** [comment]
ok glad you figures it out.
